freqtrade/templates/strategy_analysis_example.ipynb

```diff
@@ -19,9 +19,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "OperationalException",
+     "evalue": "Directory `/Users/surfer/Software/MMM/develop/freqtrade/freqtrade/templates/user_data` does not exist. Please use `freqtrade create-userdir` to create a user directory",
```

Comment:
Please make sure to only commit what you'd like to have included.
a jupyter notebook that's been executed is clearly not among these.